src/vivarium_gates_child_iv_iron/results_processing/process_results.py

```diff
@@ -28,27 +28,52 @@
 }
 
 
-def make_measure_data(data):
+def make_measure_data(data, disaggregate_seeds):
```

Comment:
Should these `disaggregate_seeds` options all default to `False` for compatibility?

Reply:
The default is False for all of these unless the flag is given when you run make_results then they all default to true.

Reply:
It could default to False here, but since it is only called in one place (the `make_results` cli call) and that place has a default to False, I think it's fine